media-sound/teamspeak-server-bin/teamspeak-server-bin-3.1.1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils multilib systemd user
+
+DESCRIPTION="TeamSpeak is software for quality voice communication via the Internet"
+HOMEPAGE="https://www.teamspeak.com/"
+SRC_URI="amd64? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_amd64-${PV}.tar.bz2 )
+	x86? ( http://ftp.4players.de/pub/hosted/ts3/releases/${PV}/teamspeak3-server_linux_x86-${PV}.tar.bz2 )"
+
+LICENSE="teamspeak3"
```

Comment:
This is bundling a lot of libraries. FWICS, it's using libc++, so you can't unbundle most of them. However, you need to explicitly list the license for every one of them.

Reply:
When I do look at ts3server, tsdnsserver and both libs, I only see glibc, zlib, openssl and mariadb-connector-c.

Looking at LICENSE:
/usr/portage/sys-libs/glibc/glibc-2.27-r1.ebuild:LICENSE="LGPL-2.1+ BSD HPND ISC inner-net rc PCRE"
/usr/portage/dev-libs/openssl/openssl-1.0.2n.ebuild:LICENSE="openssl"
/usr/portage/dev-db/mariadb-connector-c/mariadb-connector-c-3.0.3.ebuild:LICENSE="LGPL-2.1"
/usr/portage/sys-libs/zlib/zlib-1.2.11-r1.ebuild:LICENSE="ZLIB"

So I have to add at least "LGPL-2.1+ openssl ZLIB" to LICENSE?

Reply:
`LICENSE` needs to express licenses for all files that are *installed*, even if you believe they're not being used. You can ofc strip them but I kinda doubt they're really unused.

Reply:
Okay, so that would be in case LGPL-2.1 for mariadb-connector-c (for the single lib in redist/) and teamspeak3 for the Rest.